Proofs/Experiments/Memcpy/MemCpyVCG.lean

```diff
@@ -15,11 +15,13 @@ import Tactics.Name
 import Tactics.ClearNamed
 import Tactics.StepThms
 import Correctness.ArmSpec
-import Arm.Insts.Common
 import Arm.Memory.SeparateAutomation
 import Arm.Syntax
 import Tactics.SkipProof
 
+-- Disable linters, they take too much time.
```

Comment:
Sigh. Is it the unused vars check again, or something else? I believe the former's rehaul is in progress.

Reply:
Yes, the same!~

Reply:
Well then why don't we do just the following:
`set_option linter.unusedVariables false in...`

Reply:
Thanks for forcing me to be precise :) Done.